.gitignore

```diff
@@ -3,7 +3,7 @@
 .nix-shell-cabal.project
 /cabal.project.diff
 /cabal.project.freeze
-/cabal.project.local
+/cabal.project.local*
```

Comment:
Do we want `cabal.project.local` in the repo? I tend to use this file to have development-specific configuration (like disabled optimizations for faster development builds.

Reply:
You're right, it matches 0 or more.